### DISCUSSION TITLE
Compiled custom firmware shows unknown version

### DISCUSSION THREAD

**Discussion by robustini** [ExpressLRS/ExpressLRS#1772] [Ideas]
When compiling the firmware the version shown on the LUA script is rightfully unknown.
Would it be possible to put anything else in the place of unknown?
In the lua script it appears "FORMA" which does not say anything, would something like "CUSTOM" not be more appropriate?

def get_version (env):
     return '% s (% s)% s'% (env.get ('GIT_VERSION'), env.get ('GIT_SHA'), env.get ('REG_DOMAIN'))

**Comment by pkendall64** [comment]
The GIT_VERSION comes from the tag or branch if its's built from a git clone, if it's from a zip then the git information is not known we we use unknown.
In the LUA we have very little room, display the text so its shortened, but the hash part at least can point us to the right commit. Again, unless it's from a zip download in which none if the information is available and so just "unknown" is displayed.
